src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -153,6 +153,7 @@ class NetworkAppGatewayDefaultScenarioTest(ScenarioTest):
     def test_network_app_gateway_with_defaults(self, resource_group):
         self.cmd('network application-gateway create -g {rg} -n ag1 --no-wait')
         self.cmd('network application-gateway wait -g {rg} -n ag1 --exists')
+        self.cmd('network application-gateway update -g {rg} -n ag1 --no-wait')
```

Comment:
Do you need to wait before the next `update` gets executed? Or it doesn't matter as it is really a no-op?

Reply:
Yeah, no-op so doesn't really matter. The nature of updating app gateways anyways is that you do a lot of consecutive updates. It's just important that the initial `wait ... --exists` happens so you don't get the 404.